app/models/pic.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Pic < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_one_attached :image
```

Comment:
I'm not sure that your ActiveStorage db migration got saved, because it's when a pic tries to get stored that errors occur